test/searchRestaurant.test.ts

```diff
@@ -0,0 +1,43 @@
+// import axios from 'axios';
```

Comment:
please delete this file, if you will need it later to recreate tests, you can always find it in git history